src/fluree/db/query/fql/parse.cljc

```diff
@@ -1,23 +1,23 @@
 (ns fluree.db.query.fql.parse
-  (:require [fluree.db.query.exec.eval :as eval]
-            [fluree.db.query.exec.where :as where]
-            [fluree.db.query.exec.update :as update]
-            [fluree.db.query.exec.select :as select]
-            [fluree.db.datatype :as datatype]
-            [fluree.db.query.fql.syntax :as syntax]
+  (:require #?(:cljs [cljs.reader :refer [read-string]])
             [clojure.set :as set]
             [clojure.walk :refer [postwalk]]
-            [fluree.json-ld :as json-ld]
+            [fluree.db.constants :as const]
+            [fluree.db.datatype :as datatype]
+            [fluree.db.query.exec.eval :as eval]
+            [fluree.db.query.exec.select :as select]
+            [fluree.db.query.exec.update :as update]
+            [fluree.db.query.exec.where :as where]
+            [fluree.db.query.fql.syntax :as syntax]
             [fluree.db.util.core :as util :refer [try* catch*]]
             [fluree.db.util.log :as log :include-macros true]
             [fluree.db.validation :as v]
-            [fluree.db.constants :as const]
-            #?(:cljs [cljs.reader :refer [read-string]])))
+            [fluree.json-ld :as json-ld]))
 
 #?(:clj (set! *warn-on-reflection* true))
 
 (defn parse-var-name
-  "Returns a `x` as a symbol if `x` is a valid '?variable'."
+  "Returns a `?x` as a symbol if `?x` is a valid '?variable'."
```

Comment:
I think this docstring change makes things more confusing. `x` is supposed to be referring to the argument supplied to this function, which does not include the '?' character in it's name. The argument supplied is `x` and doesn't include a '?' in it's name because it may not be a valid variable.

Reply:
That's not how it behaves. If you pass `x` or `"x"` as an arg, it will return `nil`. If you pass `?x` or `"?x"`, it will return `?x`.

Reply:
`x` is the name of the argument binding this function accepts. The function does not accept an argument called `?x`; it accepts an argument called `x`. The '?variable' (note the different quotation marks) indicates that the *value* of `x` needs to begin with a '?' if this function returns anything. The `x` in this docstring refers to the literal argument name, and this change doesn't make any sense because this function doesn't accpt an argument called `?x`.

Reply:
OK I understand the confusion now. I was talking about the value of the argument and you were talking about the name.

I was confused by what values the fn expected with the original docstring, but I agree that this can be confusing in a different way now.

I have an idea for a revision that should clarify both situations.